docs/functional-components/http-clients.md

```diff
@@ -0,0 +1,68 @@
+# Clients for REST/HTTP APIs
+
+Node.js applications and frameworks often need to call existing REST/HTTP APIs
+as clients.
+
+## Recommended packages
+
+- [axios](https://github.com/axios/axios) for Node.js
+
+  Axios is a promise based HTTP client for the browser and node.js
+
+- [node-fetch](https://github.com/node-fetch/node-fetch) for standard `fetch` API compatibility
+
+  A light-weight module that brings [window.fetch](https://fetch.spec.whatwg.org/) to Node.js
```

Comment:
With 2 options I think we need to explain when we would recommend one versus the other.